CRM/Utils/Hook.php

```diff
@@ -1811,13 +1811,9 @@ public static function triggerInfo(&$info, $tableName = NULL) {
    * @param array $fields
    *   List of data fields to translate, organized by table and column.
    *   Omitted/unlisted fields are not translated. Any listed field may be translated.
-   *   Values should be TRUE-ish.
+   *   Values should be TRUE.
    *   Ex: $fields['civicrm_event']['summary'] = TRUE
-   *   Ex: $fields['civicrm_event']['summary'] = 'yesplease';
-   *
-   * At time of writing, the `$fields` list x is prepopulated based on `<localizable>` fields in core's `xml/schema`.
-   * In the future, it may also be prepopulated with `<localizable>` fields in ext's `xml/schema`.
-   * For the interim, you may wish to fill-in `<localizable>` fields from ext's.
+   * @return mixed
```

Comment:
In other places we have    * @return null
   *   the return value is ignored
Do we ignore this one?

Reply:
@eileenmcnaughton Yes, the return value is ignored here. 

Of course, both `@return mixed` and `@return null` feel misleading to me (for the ~95% of hooks which don't really return anything).  To my skimming, the `@return mixed` looks more consistent with the style of surrounding code. (~77x `mixed` vs ~35x `null`).

If we wanted it to make sense, then I suppose those ~110 regular (non-returning) hook-stubs should actually be true-void...

Reply:
hmm ok -